pymanrs/manrs.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# MANRS API - developed by acidvegas in python (https://git.acid.vegas/manrs)
+
+'''
+Source             : https://www.manrs.org/
+Request API Access : https://www.manrs.org/resources/api
+API Doumentation   : https://manrs.stoplight.io/docs/manrs-public-api
+'''
+
+import http.client
+
+class API(object):
+	def __init__(self, api_key: str, dev: bool=False):
+		'''
+		MANRS API class.
+		
+		:param api_key: Your MANRS API key
+		:param dev: Whether or not to use the development API (default: False)
+		'''
+		self.api_key = api_key
+		self.dev = dev
+
+	def call(self, endpoint: str) -> dict:
+		'''
+		Makes a call to the MANRS API
+
+		:param endpoint: The endpoint you would like to query
+		'''
+		headers = {'Accept': 'application/json', 'Authorization': 'Bearer ' + self.api_key}
+		if not self.dev:
+			conn = http.client.HTTPSConnection('api.manrs.org')
+		else:
+			http.client.HTTPSConnection('api-dev.manrs.org')
+		conn.request('GET', endpoint, headers=headers)
+		res = conn.getresponse()
+		data = res.read()
+		return data.decode()
+
+	def roa_by_asn(self, asn: str) -> dict:
+		'''
+		Retrieve data about ROAs by ASN
+		
+		:param asn: The ASN you would like to query either as a number or in AS12345 format
```

Comment:
there's a pylint line length limit of 80 chars.
(I'm not sure that pylint applies here, but.. it's sure handy!)

Reply:
Are you telling me my lines have to be 80 chars to be approved for merging, or just suggesting it?

Reply:
i'm saying that the python linter is going to complain, so it'd sure be nicer if it didn't complain.